ablySpec/RealtimeClientChannel.swift

```diff
@@ -1297,6 +1299,66 @@ class RealtimeClientChannel: QuickSpec {
 
             }
 
+            context("crypto") {
+                it("if configured for encryption, channels encrypt and decrypt messages' data") {
```

Comment:
Right now that wouldn't work, because that behavior is not working properly (see #211). Can I just intercept the raw `ProtocolMessage` and check that the data is encoded and the encoding is set there?